app/src/main/java/com/infomaniak/drive/ui/MainActivity.kt

```diff
@@ -298,6 +298,12 @@ class MainActivity : BaseActivity() {
 
         handleBottomNavigationVisibility(destination.id, shouldHideBottomNavigation)
 
+        // TODO Find a better way to do this. Currently, we need to put that here and not in the preview slider fragment
+        // because of APIs <= 27.
```

Comment:
```suggestion
        // TODO: Find a better way to do this. Currently, we need to put that
        // here and not in the preview slider fragment because of APIs <= 27.
```